### PR TITLE
Added Kudos Model

### PR DESCRIPTION
**Pull request by konrad04-1989**

### CODE REVIEW

File: app/controllers/kudos_controller.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class KudosController < ApplicationController
+  before_action :set_kudo, only: %i[show edit update destroy]
```
Comment (ciejj) [inline app/controllers/kudos_controller.rb:4]: FYI - no-one is actually using callbacks (besides authentication) - it is actually making a code harder to read.
Let's leave it like this for now

File: app/models/kudo.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Kudo < ApplicationRecord
+  validates :title, presence: true, length: { minimum: 2 }
+  validates :content, presence: true, length: { minimum: 2 }
+  validates :giver_id, presence: true
+  validates :receiver_id, presence: true
```
Comment (ciejj) [inline app/models/kudo.rb]: you don't need this - the `belongs_to` is already taking care of that by default

If you do not add  `optional: true` option to `belongs_to` it is expecting Kudo to have that association

File: app/views/kudos/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="bg-light shadow-lg p-4 mb-5 rounded">
+<h2>Create New Kudo!</h2>
+
+<%= render 'form', kudo: @kudo %>
+
+<hr class="my-4">
+<%= link_to 'Back To Kudos List', kudos_path, class:"btn btn-secondary" %>
+</div>
```
Comment (ciejj) [inline app/views/kudos/new.html.erb]: missing newline at the end of the file - you can configure your IDE to take care of it for you, when you are saving the file :)

File: app/views/kudos/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    <%= @kudo.receiver_of_kudo.email %>
+  </p>
+  <hr class="my-4">
+
+  <div class="d-grid gap-1 d-md-flex mx-left">
+    <%= link_to 'Edit This Kudo...', edit_kudo_path(@kudo), class:"btn btn-warning" %>
+    <%= button_to 'Destroy This Kudo!', @kudo, method: :delete, :onclick => "return confirm('Are you sure you want to delete this Kudo?')", class:"btn btn-danger" %>
+    <%= link_to 'Back To Kudos List', kudos_path, class:"btn btn-secondary" %>
+  </div>
+</div>
```
Comment (ciejj) [inline app/views/kudos/show.html.erb]: missing newline ;)

File: db/seeds.rb [hunk 5]
```diff
@@ -5,8 +5,49 @@
...
+employee.email = 'emp4@rera.com'
+employee.password = 'zaq123'
+employee.password_confirmation = 'zaq123'
+employee.save!
+
+employee = Employee.new
+employee.email = 'emp5@rera.com'
+employee.password = 'zaq123'
+employee.password_confirmation = 'zaq123'
+employee.save!
```
Comment (ciejj) [inline db/seeds.rb]: you used a loop to create Kudos - maybe you could use one for creating Employees as well?

File: db/seeds.rb [hunk 6]
```diff
@@ -5,8 +5,49 @@
...
+5.times do
+    kudo = Kudo.new
+    kudo.title = Faker::Superhero.name
+    kudo.content = Faker::TvShows::Friends.quote
+    kudo.giver_id = Faker::Number.between(from: 1, to: 5)
+    kudo.receiver_id = Faker::Number.between(from: 1, to: 5)
+    kudo.save!
+end
+
+puts "Kudos table seeding done"
```
Comment (ciejj) [inline db/seeds.rb]: and newline ;)

**Review by ciejj — CHANGES_REQUESTED** [review]
Really nice job!
I have suggested couple of really small changes. 
Let me know if you have any questions

Also, could you please paste the task description to the PR description? (for next PRs) - it will make the review easier :)

File: db/seeds.rb [hunk 7]
```diff
@@ -7,35 +7,16 @@
...
-employee.password = 'zaq123'
-employee.password_confirmation = 'zaq123'
-employee.save!
-
-employee = Employee.new
-employee.email = 'emp5@rera.com'
-employee.password = 'zaq123'
-employee.password_confirmation = 'zaq123'
-employee.save!
+5.times do
```
Comment (ciejj) [inline db/seeds.rb]: try this in console:
```
5.times do |i|
  puts i
end
```
and get rid of `employeenumber` :)

**Review by ciejj — APPROVED** [review]

**Review by ciejj — APPROVED** [review]

**Review by oskarlakner — APPROVED** [review]
it's so nice! I'll show you today my approach to writing seeds :v:

**Pull request merged**